payloads/library/GitBunnyGit/payload.txt

```diff
@@ -41,31 +41,41 @@ export GIT_SSL_NO_VERIFY=1
 if [ -d ".git" ]; then
 	# Get the newest payloads
 	git pull origin $git_branch &>> $log_file
+	echo "Git repository updated." >> $log_file
 	LED G 200
-	exit 1
 else
 	# Move the existing payloads directory, in case hackers be hackin'
 	mv payloads payloads-orig
+	echo "Existing 'payloads' directory preserved as 'payloads-orig'" >> $log_file
 	# Initialize Repository
 	git init &>> $log_file
 	# Let's pick the hak5 github repo
 	git remote add origin $git_repo &>> $log_file
+	echo "Git repository selected: $git_repo" >> $log_file
 	# Instead of cloning the whole repo, 
 	git config core.sparsecheckout true
+	echo "Git configuration change: sparse-checkout=true." >> $log_file
 	# isolate the payloads directory
 	echo "payloads" >> .git/info/sparse-checkout
+	echo "Sparse checkout: 'payloads' directory selected" >> $log_file
 	# "git clone"
 	git pull origin $git_branch &>> $log_file
+	echo "Git repository cloned." >> $log_file
 
-	# Ignore any existing directories or files, so git status is pretty
+	# Ignore any existing directories or files, so git status is pretty, and git pull will work after the "clone"
 	LED R B
 	for file in $(ls -A); do
-		[[ "${file}" =~ [^payloads$] ]] && echo "${file}" >> .gitignore
+		[[ "${file}" =~ [^payloads$] ]] && { echo "${file}" >> .gitignore; echo ".gitignore add: ${file}" >> $log_file; }
 	done
+	echo "payloads/switch1" >> .gitignore
```

Comment:
It doesn't seem to accept the payload.txt file after it is copied back from 'payloads-orig'